content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,93 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Improved** performance of `ServiceEntry` updates #19305.
+- **Improved**fix readiness probe inconsistency #18164.
+- **Improved** performance of configuration updates by sending partial updates where possible #18354.
+- **Added** an option to control locality load balancer settings per host #18406.
```

Comment:
Do we still link to the issues in the major version release notes? There's quite a few issue numbers here without links.

Reply:
I'll do a last pass today at following PRs to issues when possible and will remove the PR number once done.
Also will remove the PR number if there is no link to an issue. I could use some help here (under the form of review with code suggestion)